src/main/kotlin/no/nav/familie/klage/brev/domain/Avsnitt.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.klage.brev.domain
+
+import no.nav.familie.klage.felles.domain.Sporbar
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
+import java.util.UUID
+
+data class Avsnitt(
+    @Id
+    val avsnittId: UUID = UUID.randomUUID(),
+    val behandlingId: UUID,
+    val deloverskrift: String,
+    val innhold: String,
+    @Column("skal_skjules_i_brevbygger")
+    val skalSkjulesIBrevbygger: Boolean? = false,
```

Comment:
Nullable?

Reply:
Det er bare flytt av objekter, får sette den til not null senere